ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Hey team! 
+
+I was using your awesome geocoding engine when I noticed something interesting.
+Let me tell you more about it.
+
+---
+#### Here's what I did :innocent:
+_try adding a link here to either [pelias.io/compare](http://pelias.io/compare) or [mapzen.com/search](https://mapzen.com/search)_
+
+---
+#### Here's what I got :scream_cat:
+_we :heart: screenshots or copy-pasted json results_
```

Comment:
An excellent addition here would be something like "Please paste any logs you think are relevant, and please just paste the whole log: it's much better to have to wade through a lot of output than be missing the critical part"